overlord/servicestate/quota_handlers.go

```diff
@@ -218,9 +218,10 @@ func addRestartServicesTasks(st *state.State, queueTask func(task *state.Task),
 	for _, info := range sortedInfos {
 		restartTask := st.NewTask("service-control", fmt.Sprintf("Restarting services for snap %q", info.InstanceName()))
 		restartTask.Set("service-action", ServiceAction{
-			Action:   "restart",
-			SnapName: info.InstanceName(),
-			Services: getServiceNames(servicesAffected[info]),
+			Action:                  "restart",
+			SnapName:                info.InstanceName(),
+			Services:                getServiceNames(servicesAffected[info]),
+			RestartEnabledNonActive: false,
```

Comment:
here we know we want false, but I suspect that there maybe other places where we want to set this to true? where do we also create ServiceAction tasks?

Reply:
Right, I forgot to change that, I've pushed changes to fix it.